test/spec/index.js

```diff
@@ -56,6 +74,8 @@ describe('Clay', function() {
 
     it('handles the "showConfiguration" event if autoHandleEvents is not false',
     function() {
+      this.timeout(10000); // 10 seconds
```

Comment:
Have you tried logging/debugging this bit of code in this test to see where it is stopping and where the code paths take you?